starter-code/src/functions-and-arrays.js

```diff
@@ -11,30 +27,89 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+var findLongestWord = function (array){
+var lengthOfWord = 0;
+var longestWord;
+for (var i = 0; i < array.length; i++){
+  if(array[i].length > lengthOfWord){
+     lengthOfWord = array[i].length;
+     longestWord = array[i];
+  }
+}
+return longestWord
+}
+
+findLongestWord(words);
+
+ //Calculating a Sum
+
+//  This was my first solution to exercise, but the function did not return anything, it only logged it to the console
+//  var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+
+//  var sumArray = numbers.reduce(function(accumulator,currentValue){
+//  return accumulator + currentValue;
+//  },0);
+//  console.log(sumArray);
+
+ var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+ function sumOfArray(array){
+   var counter = 0;
+   for(var i = 0; i < array.length; i++){
+     counter += array[i];
+   }
+   return counter;
+ }
+ sumOfArray(numbers)
+
+ 
 
 // Calculate the Average
 
+ 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+var theSum1  = numbersAvg.reduce(function(accumulator,currentValue){
+  return accumulator + currentValue}, 0);
+
+function averageNumbers(sum,array){
+  var arrayTotal = 0;
+    for (var i = 0; i < array.length; i++){
+    arrayTotal += 1;}
+ 
+    return sum/arrayTotal;
+  }
+ 
+ averageNumbers(theSum1,numbersAvg);
 
 // Array of Strings
-var wordsArr = [
-  'seat',
-  'correspond',
-  'linen',
-  'motif',
-  'hole',
-  'smell',
-  'smart',
-  'chaos',
-  'fuel',
-  'palace'
-];
+ var wordsArr = [
+   'seat',
+   'correspond',
+   'linen',
+   'motif',
+   'hole',
+   'smell',
+   'smart',
+   'chaos',
+   'fuel',
+   'palace'
+ ];
+
 
-// Unique Arrays
-var wordsUnique = [
```

Comment:
numOfElements could be written at array.length right?

Reply:
Yes, I need to watch this, thanks for the feedback:)